Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM python:3 as base
+WORKDIR /app
+COPY ./poetry.toml /app
+COPY ./pyproject.toml /app
+RUN pip install poetry && poetry install
+
+FROM base as production
+
+RUN apt-get update && \
+    apt-get install -y gunicorn 
```

Comment:
Rather than installing gunicorn directly, we can also track that with poetry (which similarly means we can run it with the simpler `poetry run gunicorn <options>`)